src/Symfony/Component/Workflow/Registry.php

```diff
@@ -23,19 +24,26 @@ class Registry
 
     /**
      * @param Workflow $workflow
-     * @param string   $className
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $className)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
no, don't revert this one, as injecting class names directly will trigger a deprecation warning. We don't want the normal configuration of the bundle to trigger a deprecation warning for our users (as they would not be in control of fixing it anyway)